docs(file-upload): added warning for cloud providers (#1616)
nestjs/docs.nestjs.com: content/techniques/file-upload.md

- `fieldName`: string that supplies the name of the field from the HTML form that holds a file
- `options`: optional object of type `MulterOptions`. This is the same object used by the multer constructor (more details [here](https://github.com/expressjs/multer#multeropts)).

> > warning **Warning** `FileInterceptor()` is not compatible with third party cloud providers type Firebase (and other cloud providers)

Reviewer: ```suggestion
> warning **Warning** `FileInterceptor()` may not be compatible with third party cloud providers like Google Firebase or others.
```

Reviewer: Maybe we could add some more information WHY it is not compatible?

Reviewer 2: I like the language here of `may not be` over `is not`, cause there's probably ways around things. As for why they aren't compatible, it could be useful information, but to start off we can probably just throw up the warning and iterate on it as necessary
